### PR TITLE
tgif 4.2.5 (new formula)

### PR DESCRIPTION
**Pull request by fabianschuiki**
Add the tgif 4.2.5 2D drawing tool. Dates back quite a bit, but is still
being maintained. Very useful for drawing block diagrams.

### CODE REVIEW

File: Library/Formula/tgif.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Tgif < Formula
+  desc "Xlib-based interactive 2D drawing tool"
+  homepage "http://bourbon.usc.edu/tgif/"
+  url "https://downloads.sourceforge.net/projects/tgif/files/tgif/4.2.5/tgif-QPL-4.2.5.tar.gz"
+  md5 "a622240ce2377f15b6d8261e4c49b8f6"
+  sha1 "d25c8aaa7ddbeba4e720dc369d62f14b850065bc"
+
+  depends_on :x11
```
Comment (mikemcquaid) [inline Library/Formula/tgif.rb]: Should be submitted to `homebrew-x11`

File: Library/Formula/tgif.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class Tgif < Formula
+  desc "Xlib-based interactive 2D drawing tool"
+  homepage "http://bourbon.usc.edu/tgif/"
+  url "https://downloads.sourceforge.net/projects/tgif/files/tgif/4.2.5/tgif-QPL-4.2.5.tar.gz"
+  md5 "a622240ce2377f15b6d8261e4c49b8f6"
+  sha1 "d25c8aaa7ddbeba4e720dc369d62f14b850065bc"
```
Comment (mikemcquaid) [inline Library/Formula/tgif.rb]: Use `sha256` and remove the `md5` and `sha1`

File: Library/Formula/tgif.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+  depends_on :x11
+
+  def install
+    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
+    system "make", "install"
+  end
+end
```
Comment (mikemcquaid) [inline Library/Formula/tgif.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by fabianschuiki** [comment]
Fixed issues and resubmitted to `homebrew-x11`. Cheers :)

**Pull request closed without merging**